static/js/compose.js

```diff
@@ -574,7 +574,7 @@ exports.validate = function () {
 };
 
 exports.initialize = function () {
-    $('#stream,#subject,#private_message_recipient').bind({
+    $('#stream,#subject,#private_message_recipient').on({
```

Comment:
yeah, agreed. Kind of forgot to post about the agreement part later when I checked out the official docs page in detail and didn't find anything.